CONTRIBUTING.md

```diff
@@ -153,66 +180,127 @@ Sometimes, it might not make sense for ECS to standardize a field, value, or cat
-index (multi, comma separated): winlogbeat-*
-language [kuery] ("n/a" to leave blank)  (required): kuery
-query (required): event.category:process and process.parent.name:msbuild.exe
-ecs_version (multi, comma separated): 1.4.0
+add mitre tactic? [y/N]:
+index (multi, comma separated):
+language [kuery] ("n/a" to leave blank)  (required):
+query (required):
+author (required)  (multi, comma separated):
+building_block_type:
+exceptions_list (multi, comma separated):
+license [Elastic License] ("n/a" to leave blank)  (required):
+risk_score_mapping (multi, comma separated):
+rule_name_override:
+severity_mapping (multi, comma separated):
+timestamp_override: N
+mitre tactic name (required):
+technique IDs for Command and Control (required)  (multi, comma separated):
+add mitre tactic? [y/N]:
 ```
 
-Pending no errors, you'll see this output upon success
+1. You will receive this output when successful:
+
 ```
-Rule Suspicious Child of MsBuild saved to rules/windows/defense_evasion_msbuild_child.toml
+Rule saved to /PATH/TO/detection-rules/RULE.toml
 Did not set the following values because they are un-required when set to the default value
  - from
  - interval
  - max_signals
  - to
+Rule will validate against the latest ECS schema available (and beats if necessary)
+    - to have a rule validate against specific ECS schemas, add them to metadata->ecs_versions
+    - to have a rule validate against a specific beats schema, add it to metadata->beats_version
 ```
 
 
-### Testing a rule with the CLI
+### Importing rules
+
+1. As shown above, rules can be created in an appropriate directory with `rules/`. ***If*** rules are created outside of this structure, you can import them into the project repo with the `import-rules` sub-command.
+
+1. For example, this command will copy any newly-developed rule files into `rules/` directory:
+
+```shell
+    python -m detection_rules import-rules -d ~/dev-rules/
+```
+
+### Linting and validation
+
+After import, you can check the syntax and validate new rules.
+
+1. Example linting:  
+python -m detection_rules toml-lint -f rules/telnet_port_activity.toml
+
+  > The CLI will highlight any syntax issues in your toml.
+
+1. After the rule’s syntax is checked, validate the rule by running the validate-rule command. Provide the path to the rule, followed by the rule UID.
+python -m detection_rules validate-rule -p rules/telnet_port_activity.toml 4a4e23cf-78a2-449c-bac3-701924c269d3
+
+
+### Testing rules
 
 When a rule is ready, it can be tested with unit tests. Detection Rules has several tests that run locally to validate rules in the repository. These tests make sure that rules are syntactically correct, use ECS or Beats schemas correctly, and ensure that metadata is also validated. There are also internal tests to make sure that the tools and functions to manager the repository are working as expected.
 
 To run tests, simply run the command `test` with the CLI
-```console
+
+```shell
```

Comment:
Anything wrong with `console`? There are a lot more places than just here where we use it

Reply:
Take _that_ @dcode, we love blue ;)